codegen/templates/code/controller/event_handlers.gotmpl

```diff
@@ -62,17 +64,17 @@ type {{ $resource.Kind }}EventWatcher interface {
     AddEventHandler(ctx context.Context, h {{ $resource.Kind }}EventHandler, predicates ...predicate.Predicate) error
```

Comment:
Shouldn't we remove predicates from this interface?

Reply:
nope, they are handled by creating a new controller each time